Week02/lesson03/homework/relaxr_starter_code/css/style.css

```diff
@@ -0,0 +1,225 @@
+/*
+Project Name: Blank Template
+Client: Your Client
+Author: Your Name
+Developer @GA in NYC
+*/
+
+
+/******************************************
+/* SETUP
+/*******************************************/
+
+/* Box Model Hack */
+* {
+     -moz-box-sizing: border-box; /* Firexfox */
+     -webkit-box-sizing: border-box; /* Safari/Chrome/iOS/Android */
+     box-sizing: border-box; /* IE */
+}
+
+/* Clear fix hack */
+.clearfix:after {
+     content: ".";
+     display: block;
+     clear: both;
+     visibility: hidden;
+     line-height: 0;
+     height: 0;
+}
+
+.clear {
+  clear: both;
+}
+
+.alignright {
+  float: right;
+  padding: 0 0 10px 10px; /* note the padding around a right floated image */
+}
+
+.alignleft {
+  float: left;
+  padding: 0 10px 10px 0; /* note the padding around a left floated image */
+}
+
+/******************************************
+/* BASE STYLES
+/*******************************************/
+
+body {
+     color: #000;
+     font-size: 12px;
+     line-height: 1.4;
+     font-family: 'Open Sans', sans-serif;
+     rgb(134,141,157);
+
+}
+
+
+/******************************************
+/* LAYOUT
+/*******************************************/
+
+/* Center the container */
+#container {
+  width: 100%;
+  margin: auto;
+
+
+}
+
+/*I stuggled to make the layout work with the bk img in the body so had to move it into the header!*/
```

Comment:
And you were a 100% right to do so! That first part with just the image and the call to action ("Get piece of mind with a single tap", "Get it now") could easily be considered a `<header>`. Also, if you apply the background image to the `<body>` then it'll cover the full width of the screen all the time, whereas by having it be the `background` of the header, then the `header` will be as tall as the image but you can scroll down to sections that don't have the image as their background.

For example if you go here: https://css-tricks.com/examples/FullPageBackgroundImage/progressive.php you'll see that the page's background image never moves or changes, the code shows us the background image has been added to the `html` tag:

![image](https://cloud.githubusercontent.com/assets/5670279/17310082/21282c4c-583a-11e6-99f6-884db8c7cf91.png)

.... but this "never-changing" image is NOT the effect we want for Relaxr, so good job only appolying the bg image to the header.